Update nf-imagehlp-getimageconfiginformation.md

bekir-ozturk opened the pull request:
Param `ImageConfigInformation` description claims that the type of the parameter is `IMAGE_LOAD_CONFIG_DIRECTORY64` when in fact it is `IMAGE_LOAD_CONFIG_DIRECTORY` which may be either `IMAGE_LOAD_CONFIG_DIRECTORY32` or `IMAGE_LOAD_CONFIG_DIRECTORY64` depending on the platform architecture.

I updated the link to point to the 64-bit version thinking it was preferrable to the 32-bit one.

bekir-ozturk committed:
Update nf-imagehlp-getimageconfiginformation.md

Param `ImageConfigInformation` description claims that the type of the parameter is `IMAGE_LOAD_CONFIG_DIRECTORY64` when in fact it is `IMAGE_LOAD_CONFIG_DIRECTORY` which may be either `IMAGE_LOAD_CONFIG_DIRECTORY32` or `IMAGE_LOAD_CONFIG_DIRECTORY64` depending on the platform architecture.

PRMerger-2 commented:
@bekir-ozturk : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Karl-Bridge-Microsoft committed:
Update nf-imagehlp-getimageconfiginformation.md

Updated to provide clarification without linking to new page and moved see also links up to relevant param.

PRMerger13 commented:
@Karl-Bridge-Microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.
